tiles/src/main/java/com/protomaps/basemap/layers/Earth.java

```diff
@@ -29,6 +30,19 @@ public void processNe(SourceFeature sf, FeatureCollector features) {
     } else if (sourceLayer.equals("ne_10m_land")) {
       features.polygon(this.name()).setZoomRange(5, 5).setBufferPixels(8).setAttr("kind", "earth");
     }
+    if (sourceLayer.equals("ne_10m_glaciated_areas")) {
+      try {
+        Point centroid = (Point) sf.centroid();
+        if (centroid.getY() > 0.7) {
```

Comment:
This works, but it's confusing this isn't a negative number, and 70 would normally not result in the northern tip of Chilean Antarctica (many international bases there) showing up correctly as ice.

Please add a comment explaining that you're targeting ice features in Antarctica south of -60°.

Reply:
Web Mercator Y in Planetiler goes from 0 to 1 from 85 deg N to 85 deg S. The equator is at Web Mercator Y = 0.5

Reply:
Added the comment